app/main/views.py

```diff
@@ -1,7 +1,21 @@
 from flask import render_template
 from . import main
+from ..models import Resource
+import json
+from flask import jsonify
 
-
-@main.route('/')
+@main.route('/', methods=['GET'])
 def index():
     return render_template('main/index.html')
+
+@main.route('/get-resource', methods=['GET'])
+def getResource():
+    Resource.generate_fake()
```

Comment:
This probably shouldn't be here, right? I think we'd first call `add_fake_data` (which would call `Resource.generate_fake()`).